tests/ui/imports/issue-115380.rs

```diff
@@ -0,0 +1,10 @@
+// check-pass
+// compile-flags: --emit link
```

Comment:
`--emit link` is already the default, no need to specify it.

Reply:
When using `﻿./x.py test`, the default value of ﻿`--emit` might be `﻿metadata`. see: https://github.com/rust-lang/rust/blob/master/src/tools/compiletest/src/runtest.rs#L3707-L3708.

Reply:
If the issue doesn't reproduce with just checking, and the goal is to continue compilation further then the test should probably use `// build-pass` instead of `// check-pass` and specifying `--emit link` manually.
https://rustc-dev-guide.rust-lang.org/tests/ui.html#controlling-passfail-expectations